src/random.ts

```diff
@@ -0,0 +1,3 @@
+export function randBetween(low:number, high:number){
+    return (Math.floor(Math.random()*(high-low+1)))+low;
+}
```

Comment:
Would prefer this to be a static method on a class, for neater typescript formatting.